luigi/parameter.py

```diff
@@ -537,6 +550,19 @@ def normalize(self, value):
     def _parser_action():
```

Comment:
Seems like it can be since the only place it is overridden is in BoolParameter which does not even call `_parser_action` any more in `_parser_add_arguments`.